config/database.yml

```diff
@@ -5,13 +5,16 @@
 #   gem "sqlite3"
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: react_rails_app
+  hostname: localhost
+  username: postgres
+  password: postgres
 
 # Warning: The database defined as "test" will be erased and
```

Comment:
- [ ]   Based on the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#project-requirements) please make sure that your project has a Postgres database set up, currently line `22 - 28` of your `database.yml` file shows that you are using SQLite

-  I set up Postgres locally and your app worked properly 🎉

- [ ]  Kindly take out all the SQLite configuration from your configuration and ensure your app uses Postgres 👍🏿